gpytorch/kernels/grid_interpolation_kernel.py

```diff
@@ -97,18 +107,18 @@ def __init__(self, base_kernel, grid_size, num_dims=None, grid_bounds=None, acti
         self.register_buffer("has_initialized_grid", torch.tensor(has_initialized_grid, dtype=torch.bool))
 
     def _create_grid(self):
-        grid = torch.zeros(self.grid_size, len(self.grid_bounds))
+        grid = [torch.zeros(s) for s in self.grid_sizes]  # ID: now a jagged array of zeros
```

Comment:
Unnecessary to initialize tensors here. This can just be an empty list and append tensors in the for loop.